cogs/study_groups.py

```diff
@@ -155,21 +155,25 @@ async def group_create(self, ctx: Context, name: str, semester: int = 1, color:
         active_db = PrimitiveMongoData(CollectionEnum.GROUP_ACTIVE)
         guild: Guild = ctx.guild
 
-        await color_db.insert_one({name: color})
-        await active_db.insert_one({name: True})
+        if not await color_db.find_one({name: {"$exists": True}}):
+            await color_db.insert_one({name: color})
+        if not active_db.find_one({name: {"$exists": True}}):
+            await active_db.insert_one({name: True})
```

Comment:
you only need to check one database since they are mirrors from each other

Reply:
Rejected.
Each entry is independent of another and must be checked individually.
It does not matter that they are created together.